src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,39 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        String[] salaries = new String[names.length];
+        int index = 0;
+        int income = 0;
+        final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
[checklist ](https://github.com/mate-academy/jv-salary-info/blob/master/checklist.md#if-you-create-a-formatter-make-it-a-constant-field) issue -  should be constant
what you have here is just a final local variable, it's not static